src/runtime_src/core/pcie/windows/device_windows.cpp

```diff
@@ -835,6 +835,83 @@ struct rom
   }
 };
 
+struct data_retention
+{
+  using result_type = uint32_t;
+  using value_type = uint32_t;
+  
+  static result_type
+  user_get(const xrt_core::device* device)
+  {
+	  return 0;
+  }
+
+  static result_type
+  mgmt_get(const xrt_core::device* device)
+  {
+    auto init_ret = [](const xrt_core::device* dev) {
+    uint32_t ret;
```

Comment:
Formatted issue....need to indent the body of this lamda function.

Reply:
fixed